Makefile

```diff
@@ -3,6 +3,11 @@
 NOJESSIE ?= 1
 NOSTRETCH ?= 0
 
+PLATFORM_PATH := platform/$(if $(PLATFORM),$(PLATFORM),$(CONFIGURED_PLATFORM))
+PLATFORM_CHECKOUT := platform/checkout
+PLATFORM_CHECKOUT_FILE := $(PLATFORM_CHECKOUT)/$(PLATFORM).ini
+PLATFORM_CHECKOUT_CMD := $(shell if [ -f $(PLATFORM_CHECKOUT_FILE) ]; then PLATFORM_PATH=$(PLATFORM_PATH) j2 $(PLATFORM_CHECKOUT)/template.j2 $(PLATFORM_CHECKOUT_FILE); fi)
```

Comment:
To support azp, please allow the step can be skipped when the repo has already checked out.

Reply:
> To support azp, please allow the step can be skipped when the repo has already checked out.

@xumia would the change in `platform/checkout/template.j2` be sufficient to support azp?

Reply:
This step will automatically be skipped if the `$(PLATFORM_PATH)` directory already exists in the workspace